app/lib/constants.js

```diff
@@ -2,6 +2,11 @@
 module.exports = {
   ASSETS_URL: 'https://assets.nhs.uk',
   SITE_ROOT: '/find-a-chlamydia-test',
+  PROM_HISTOGRAM_BUCKETS: [0.01, 0.05, 0.1, 0.2, 0.3, 0.5, 1, 1.5, 5, 10],
```

Comment:
I thought the discussion was to not use capitalised constants?

Reply:
Will change the 2 back as per the consensus.